distribution/lib/Standard/Base/0.0.0-dev/src/Meta.enso

```diff
@@ -14,10 +14,36 @@ import project.Nothing.Nothing
 import project.Polyglot.Java
 
 import project.Error.Error as Base_Error
-import project.Polyglot.Polyglot as Base_Polyglot
+import project.Polyglot as Base_Polyglot
 
 from project.Data.Boolean import Boolean, True, False
 
+type Type
+    ## UNSTABLE
+       ADVANCED
+
+       Type meta-representation.
+
+       Arguments:
+       - value: The value of the type in the meta representation.
+    Value value
+
+    ## UNSTABLE
+       ADVANCED
+
+       Returns a vector of `Meta.Constructor` for this type
+    constructors : Vector
+    constructors self =
+        Vector.from_polyglot_array (get_type_constructors self.value Meta.Constructor.Value)
+
+    ## UNSTABLE
+       ADVANCED
+
+       Returns a vector of `Meta.Constructor` for this type
```

Comment:
is this description correct?

Reply:
Yes, it should be. `Vector[Meta.Constructor]`.